src/app/versioning/version.service.ts

```diff
@@ -78,7 +73,7 @@ export class VersionService {
 
     private notificationExists(): boolean {
         let notifications: Notification[];
-        let sub = this._notificationService.notifications.first().subscribe(n => {
+        let sub = this._notificationService.notifications.pipe(first()).subscribe(n => {
```

Comment:
This looks like a legacy code. For safety, please check if we need to call unsubscribe or not.